editions/test/tiddlers/tests/test-widget.js

```diff
@@ -805,6 +822,350 @@ describe("Widget module", function() {
 		// the <<qualify>> widget to spit out something different.
 		expect(withA).toBe(withoutA);
 	});
+
```

Comment:
I wonder if these tests should go in their own file. I added a test-action-widgets.js file recently, but this is really a hybrid. I think it would make sense to make a new `test-checkbox-widget.js` file.

Reply:
Sure. The tests were getting long enough, that even by refactoring them to get as much shared code de-duplicated as I could, they were starting to be a significant percentage of the file. I'll have to duplicate a little bit of the `test-widget` code, specifically the helper functions, but I agree that it makes sense for the checkbox tests to be in their own file.

Reply:
Done.